### DISCUSSION TITLE
Ask some questions about the use of SUMO routing

### DISCUSSION THREAD

**Discussion by radiateboy** [eclipse/mosaic#223] [Q&A]
1.  Whether SUMO rou.xml can be retained for direct use in sumocfg？
2. There are different types of definitions in the routing file. Which types are mainly supported, such as:
    `<trip id="sumo_bus_1" depart="18.39" from="2754" to="-75" type="alltype" fromTaz="taz_5" toTaz="taz_22" departLane="free" departSpeed="max"/>`

   `<trip id="sumo_obu_1" depart="0.3" type="alltype" from="-2984" to="1359" />`

`
<route repeat="900" id="r_0" probability="0.5" edges="759959733#6 -759959732#3 -759959732#2 -759959732#1"/>
`

`<routeDistribution id="allroute">
<route id="rou_1" probability="0.3" edges="1323 2790 311 2789 32">
<route id="rou_2" probability="0.5" edges="326 2417 2992 1346">
<route id="rou_3" probability="0.3" edges="2790 311 2789 326">
<routeDistribution/>
`

**Comment by kschrab** [comment]
Please follow our tutorial on converting existing SUMO scenarios to MOSAIC: https://www.eclipse.org/mosaic/tutorials/integrate_existing_sumo_scenarios/
